tests/Client/Stripe/StubCustomerService.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace BigGive\Identity\Tests\Client\Stripe;
+
+use Stripe\Customer;
+
+class StubCustomerService
+{
+    public function create(array $_params): Customer
+    {
+        $this->pause();
+
+        /** @var string $customerJSON */
+        $customerJSON = file_get_contents(
+            dirname(__DIR__, 2) . '/MockStripeResponses/customer_no_credit.json'
+        );
+        /** @var array $customer */
+        $customer = json_decode($customerJSON, true, 512, JSON_THROW_ON_ERROR);
+        $customer['id'] = 'cus_' . bin2hex(random_bytes(12));
+
+        $customer = Customer::constructFrom($customer);
+
+        \assert($customer instanceof Customer);
+
+        return $customer;
```

Comment:
```suggestion
        return $customer Customer::constructFrom($customer);
```

Psalm says the assert here is redundant.

Reply:
Makes sense, addressed locally as I think the suggestion has a typo